solidity/test/zeto_anon.ts

```diff
@@ -64,7 +61,7 @@ describe("Zeto based fungible token with anonymity without encryption or nullifi
     const tx = await erc20.connect(deployer).mint(Alice.ethAddress, 100);
     await tx.wait();
     const balance = await erc20.balanceOf(Alice.ethAddress);
-    expect(balance).to.equal(100);
+    expect(balance).to.be.gte(100);
```

Comment:
I feel we shouldn't sacrifice the scope of assertions in the tests when running against a hardhat network.

Reply:
So if the smart contract accidentally doubled the amount in the transfer logic, this check will throw an error because it knows it's running in Hardhat mode, therefore, `gte` need to be switched to `equal`?

Reply:
not sure I understand the question @Chengxuan ?

Reply:
I think the best way to address the concern of missing implementation errors like you pointed out before, without sacrificing public chain testability (where the ERC20 contract is re-used across the test suites due to ignition caching), is querying for the starting balance and ending balance around the Zeto operation. This way we can change back the check to `equal`.